RadarSDK/RadarContext.h

```diff
@@ -0,0 +1,68 @@
+//
+//  RadarContext.h
+//  RadarSDK
+//
+//  Copyright © 2020 Radar Labs, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+#import "RadarGeofence.h"
+#import "RadarPlace.h"
+#import "RadarRegion.h"
+
+/**
+ TODO(coryp): RadarContext description and documentation reference
+
+ @see https://radar.io/documentation
+ */
+@interface RadarContext : NSObject
+
+/**
+ A boolean indicating whether the location context was generated with your live API key.
+ */
+@property (assign, nonatomic, readonly) BOOL live;
+
+/**
+ An array of the geofences the location is in. May be `nil` or empty if the location is not in any geofences.
+ 
+ @see https://radar.io/documentation/geofences
+ */
+@property (nullable, copy, nonatomic, readonly) NSArray<RadarGeofence *> *geofences;
+
+/**
+ The place a location is at. May be `nil` if the location is not at a place, or if Places is not enabled.
+ 
+ @see https://radar.io/documentation/places
+ */
+@property (nullable, copy, nonatomic, readonly) RadarPlace *place;
```

Comment:
any danger that we may be overlapping / nested place? like xxx Mall / yyy store

Reply:
hm not sure what you mean, like if the backend has multiple places at the location?

Reply:
the context response is only going to show the top ranked place - we're not supporting `alternatePlaces` just yet